nipy/algorithms/group/parcel_analysis.py

```diff
@@ -267,7 +268,7 @@ def __init__(self, con_imgs, parcel_img, parcel_info=None,
         # the input full-width-at-half-maximum parameter given in mm
         # to standard deviation in voxel units.
         orient = io_orientation(self.affine)[:, 0].astype(int)
-        voxsize = np.abs(self.affine[(orient, range(3))])
+        voxsize = np.abs(self.affine[orient, :3])
```

Comment:
Oh dear - this change causes a change in the indexing, for reasons I am unable to understand or explain.  This version returns the 3 x 3 top corner of the affine, instead of a (3,) vector that is the intention.  Nasty.  Reverting.



http://docs.scipy.org/doc/numpy/reference/arrays.indexing.html#advanced-indexing